.changeset/new-tips-shop.md

```diff
@@ -0,0 +1,5 @@
+---
+'embedded-techdocs-app': patch
```

Comment:
Hmm... This is kinda interesting. We don't package/release this, but it gets bundled into `@techdocs/cli`, which we would want to get a patch bump with this.

Sufficient just to modify this changeset to use that package name, @Rugvip?

Reply:
hmmm.. on that topic https://github.com/backstage/backstage/pull/8514/files#diff-8a4003f05fedef135970d6d4cff890da5e06113ebbdc9b75b003b419835d8a72

Reply:
Ah alright, good to know, and yeah that'll do

Reply:
https://github.com/backstage/backstage/pull/8611#issue-1087617182 :grin:

Reply:
Changed 👍